sdk/highlight-go/highlight.go

```diff
@@ -255,6 +255,20 @@ func SetProjectID(id string) {
 	projectID = id
 }
 
+func SetLogOut(print bool) {
+	logOut = print
+}
+
+func Print(message string) {
```

Comment:
Per the above, I would expect we'd set the config in the logging sdk, not here. Which would mean, we don't need this wrapper around `fmt.Print`

Reply:
yup, makes sense to move this and the above logic to the `hlog` package